docker/Dockerfile_stack

```diff
@@ -234,6 +234,16 @@ RUN ["/bin/bash","-c","export KOKKOS_TOOLS_HASH=543c144bc64feb8b8cc4d8e05d04c0a0
 
 ENV KOKKOS_TOOLS_DIR=/scratch/source/kokkos-tools/release/src/tools
 
+# Benchmark support library
+ENV BENCHMARK_DIR=/opt/benchmark
+RUN cd ${PREFIX} && \
+    git clone https://github.com/google/benchmark.git -b v1.4.1 && \
+    cd benchmark && \
+    git clone https://github.com/google/googletest.git -b release-1.8.1 && \
+    mkdir build && cd build && \
+    cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=${BENCHMARK_DIR} .. && \
+    make -j${NPROC} && make install && \
```

Comment:
@Rombur Note that the `compare.py` tool is not copied to the installation prefix.  We will have to handle it ourselves if we decide to use it.

Reply:
@sslattery Look at these instructions to install the benchmark support library.  And when you configure DTK/Trilinos, pass `-D CMAKE_PREFIX_PATH=<benchmark install prefix>`